elide-datastore/elide-datastore-aggregation/src/test/java/com/yahoo/elide/datastores/aggregation/integration/MetaDataStoreIntegrationTest.java

```diff
@@ -200,10 +200,10 @@ public void tableTest() {
         // Verify Table Arguments
         given()
                 .accept("application/vnd.api+json")
-                .get("/table/SalesNamespace_orderDetails?include=arguments")
+                .get("/table/SalesNamespace_orderDetails?include=argumentDefinitions")
```

Comment:
The contract of the API shouldn't change here (should still be argument) even though we changed the model name.  The `@Include` type is still 'argument'.  Not sure why this test is passing.